### DISCUSSION TITLE
My pages default domain name became a .me domain name instead of the previous pages.dev subdomain.

### DISCUSSION THREAD

**Discussion by xdanielf** [community/community#141969] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I got a .me domain name through the GitHub student package, and my pages default domain name became a .me domain name instead of the previous pages.dev subdomain. How can I fix it?
![Screenshot 2024-10-18 233317](https://github.com/user-attachments/assets/dd2d0df9-2795-4e15-af7a-ae35c06ca99e)

**Comment by VishalKotekar** [comment]
The issue you’re encountering is because GitHub automatically assigns the `.me` domain from the GitHub Student Developer Pack when one is claimed. To revert to the default `.pages.dev` subdomain:

1. Go to your repository settings.
2. Under the "Pages" section, uncheck "Custom Domain."
3. Save the changes.

This will revert your GitHub Pages site back to the `.pages.dev` domain. Let me know if you need more details!

**Reply by LeNinjaHD** [reply]
I cant find where to uncheck this box. I only see the form to enter a new custom domain.
